mmv1/README.md

```diff
@@ -0,0 +1,38 @@
+# MMV1
+
+## Overview
+
+Mmv1 is a code generator that implements the Terraform Google Provider (TGP) resources from ruby scripts.
```

Comment:
```suggestion
Mmv1 is a code generator that implements Terraform Provider Google (TPG) resources from ruby scripts.
```

nit: TPG > TGP generally, named for the https://github.com/hashicorp/terraform-provider-google repo. Terraform Google Provider sounds better, admittedly, I typically call it "the `google` provider" when not using TPG.